.github/workflows/continuous-integration.yml

```diff
@@ -14,18 +14,6 @@ jobs:
               uses: ./.github/actions/make_install_action
             - name: Run linting
               run: npm run lint
-    postman_testing:
-        name: Running postman tests
-        runs-on: ubuntu-latest
-        steps:
-            - name: Checkout
-              uses: actions/checkout@v3
-              with:
-                fetch-depth: 0  # This causes all history to be fetched, which is required for calculate-version to function
-            - name: Perform Setup
-              uses: ./.github/actions/make_install_action
-            - name: Run tests
-              run: npm run postman-test
```

Comment:
please move the tests to the CD Github workflow instead of removing it completely

Reply:
Slight issue, I'm not sure how. There isn't a file for a CD workflow, and I'm assuming that naming it `continuous-deployment.yml` isn't going to cause it to automatically be picked up. I'll check with API-M support and see what they say.

Reply:
Any update on this? Just to be sure to have linting and postman tests running